sdk/test/trace/batch_span_processor_test.cc

```diff
@@ -211,6 +234,18 @@ TEST_F(BatchSpanProcessorTestPeer, TestManySpansLoss)
 
   // Span should be exported by now
   EXPECT_GE(max_queue_size, spans_received->size());
+
+  // At least one log message warning about the dropped span should have been emitted by now.
+  auto &vec = dynamic_cast<MockLogHandler *>(log_handler.get())->messages;
```

Comment:
Should we use `static_cast` here?Sometimes we will compile otel-cpp without RTTI.(With `-fno-rtti` or `/GR-`)

Reply:
Thanks for pointing this out, CI was also failing because of this.